x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/use_saved_search.ts

```diff
@@ -36,19 +36,32 @@ export function useSavedSearch() {
   const { currentSavedSearch, currentDataView, kibanaConfig } = mlContext;
 
   const getQueryData = () => {
-    let qry: estypes.QueryDslQueryContainer = {};
+    let qry: any = {};
     let qryString;
 
     if (currentSavedSearch !== null) {
-      const { query } = getQueryFromSavedSearchObject(currentSavedSearch);
+      const { query, filter } = getQueryFromSavedSearchObject(currentSavedSearch);
       const queryLanguage = query.language;
       qryString = query.query;
 
       if (queryLanguage === SEARCH_QUERY_LANGUAGE.KUERY) {
         const ast = fromKueryExpression(qryString);
         qry = toElasticsearchQuery(ast, currentDataView);
+        const filterQuery = buildQueryFromFilters(filter, currentDataView);
```

Comment:
I'm getting an error in the wizard when I use a saved search with a filter which uses a 'between' operator:

![image](https://user-images.githubusercontent.com/7405507/164456043-512b1be9-6a8d-4ad8-b6dd-82c4be51d7f9.png)

Gives this in the wizard:

![image](https://user-images.githubusercontent.com/7405507/164456137-45cfad72-dae4-46fe-b13a-b99bc31225cb.png)

Reply:
Actually looks like the error occurs when the saved search has a filter but nothing in the query part.

Reply:
Confirmed this is fixed by https://github.com/elastic/kibana/pull/130744/commits/f40fb58c5650f9c193bca6aa92f977e92bd41236